CHANGELOG.md

```diff
@@ -2,20 +2,20 @@
 
 ## Unversioned
 
-- Major: Added live emote updates for BTTV (#4147)
-- Minor: Added option to highlight your own messages in Highlights page under Users tab. (#3833)
-- Minor: Change the highlight order to prioritize Message highlights over User highlights. (#4303)
-- Minor: Added ability to negate search options by prefixing it with an exclamation mark (e.g. `!badge:mod` to search for messages where the author does not have the moderator badge). (#4207)
+- Major: Added live emote updates for BTTV. (#4147)
+- Minor: Added setting to turn off rendering of reply context. (#4224)
+- Minor: Changed the highlight order to prioritize Message highlights over User highlights. (#4303)
+- Minor: Added a setting to highlight your own messages in `Highlights -> Users`. (#3833)
+- Minor: Added the ability to negate search options by prefixing it with an exclamation mark (e.g. `!badge:mod` to search for messages where the author does not have the moderator badge). (#4207)
 - Minor: Search window input will automatically use currently selected text if present. (#4178)
-- Minor: Cleared up highlight sound settings (#4194)
+- Minor: Grouped highlight sound columns together and improved wording for the default sound setting. (#4194)
 - Minor: Tables in settings window will now scroll to newly added rows. (#4216)
-- Minor: Added link to streamlink docs for easier user setup. (#4217)
-- Minor: Added setting to turn off rendering of reply context. (#4224)
 - Minor: Added setting to select which channels to log. (#4302)
+- Minor: Added channel name to /mentions log entries. (#4371)
+- Minor: Added link to streamlink docs for easier user setup. (#4217)
 - Minor: Added support for HTTP and Socks5 proxies through environment variables. (#4321)
-- Minor: Remove sending part of the multipart emoji workaround (#4361)
+- Minor: Removed sending part of the multipart emoji workaround. (#4361)
```

Comment:
I know it's been merged and all but I don't want to open a new issue for it. I'm wondering whether this entry wouldn't be better suited under the "Dev" category?

Reply:
That would be fine by me